netci.groovy

```diff
@@ -4,6 +4,8 @@
 import jobs.generation.Utilities;
 def project = GithubProject
 
+buildTimeLimit = 120
```

Comment:
I don't think so. Without `def`, `buildTimeLimit` is essentially a global variable, and so available within the method `addWrappers`. Without `def`, `buildTimeLimit` would only be available within its declaration scope`.

Or at least, that's how I read the [StackOverflow article](http://stackoverflow.com/questions/184002/groovy-whats-the-purpose-of-def-in-def-x-0). I know hardly anything about groovy, so let me know if I'm wrong about this.